packages/language-service/src/completions.ts

```diff
@@ -28,6 +28,9 @@ const hiddenHtmlElements = {
   link: true,
 };
 
+const ANGULAR_ELEMENTS: ReadonlyArray<string> =
+    ['ng-container', 'ng-content', 'ng-template'] as const ;
```

Comment:
```suggestion
    ['ng-container', 'ng-content', 'ng-template'] ;
```

I don't think the cast is needed

Reply:
I guess you are right.